src/components/NavBar/_NavBar.scss

```diff
@@ -0,0 +1,173 @@
+.nav-bar{
+	width: 100%;
```

Comment:
Identation is wrong.

Reply:
Hi, so i've been checking the navbar scss file and i don't understand why git hub shows to you more spaces  that i do have, for example:
![image](https://user-images.githubusercontent.com/33492742/35833428-37226756-0a9f-11e8-8578-6ed99e4c8462.png)
As you can see, the identation is good, what should i do?